js/templates/modals/editListing/editListing.html

```diff
@@ -86,16 +92,16 @@ <h1 class="h3 txUnb padMd js-listingHeading"><% print(ob.createMode ? ob.polyT('
                 <label>Mature Content (nsfw, adult, 18+, etc)</label>
                 <% if (ob.errors['listing.item.nsfw']) print(ob.formErrorTmpl({ errors: ob.errors['listing.item.nsfw'] })) %>
                 <div class="btnStrip marginTopAuto">
-                  <div class="btn clrBr">
+                  <div class="btnRadio clrBr">
                     <input type="radio"
                            name="listing.item.nsfw"
                            value="true"
-                           id="editListingNSFWInputTrue"
+                           id="moo"
```

Comment:
Just debugging something. I believe I fixed this in the more-store branch I'm working on now. Anyhow, I'll fix here so clicking on the checkbox label works correctly.